packages/fxa-content-server/app/scripts/views/post_verify/password/force_password_change.js

```diff
@@ -6,14 +6,14 @@
  * This view handles forcing a user to change their password.
  */
 import Cocktail from 'cocktail';
+import Template from 'templates/post_verify/password/force_password_change.mustache';
```

Comment:
Prettier seems to have swapped a lot of these around and I'm not sure why... Wish we had `prettier-plugin-organize-imports` already, need to file tickets for that.

Reply:
Yeah it seems to be sorting now based on paths and I'm not sure why. I keep meaning to file the prettier-sort issues, there's just so many of them. Maybe I'll just do it today, it bothers me that my diffs keep showing sorted imports when I'm not even sure why they're being sorted and I'm pretty sure that plugin sorts by import name first. Guess it's not a huge deal if these are all going to get touched when we do that though.